packages/ui/internal/localized-fonts/localized-fonts.js

```diff
@@ -2,19 +2,21 @@ const fontMap = {};
 let currentLocale = null;
 
 // Generate a single font-face rule
-const buildFont = function (inOptions) {
```

Comment:
How will we know if this is input or output without the `in` prefix? ;)

Reply:
:lol:

Reply:
I originally wrote this back in Enyo and this was a naming convention that we used before Enyo 2.0 (more than 6 years ago), started by someone who had left before I started. It was a small callback to him/that, but it's really dated looking now and seems like a good time to change it.

_(This comment can act as that reference going forward.)_ ;)